packages/go/dawgs/drivers/neo4j/driver.go

```diff
@@ -135,6 +135,10 @@ func (s *driver) AssertSchema(ctx context.Context, schema graph.Schema) error {
 	return assertSchema(ctx, s, schema)
 }
 
+func (s *driver) SetDefaultGraph(ctx context.Context, schema graph.Graph) error {
+	return nil
```

Comment:
something sniffs funny here, I think I need context

Reply:
Ah, yeah, this needs an inline comment. Neo4j doesn't support graph namespaces so this is literally a `nop`. Will add documentation.